2019-11-20 17:00:21,384 file.manager INFO 1.2.0
2019-11-20 17:00:21,384 file.manager INFO Copyright (C) 2003-2010
2019-11-20 17:00:21,384 file.manager INFO   Noriaki Ando
2019-11-20 17:00:21,385 file.manager INFO   Intelligent Systems Research Institute, AIST
2019-11-20 17:00:21,385 file.manager INFO Manager starting.
2019-11-20 17:00:21,385 file.manager INFO Starting local logging.
2019-11-20 17:00:21,391 file.manager.namingmanager INFO NameServer connection succeeded: corba/localhost
2019-11-20 17:00:23,417 file.ManagerServant INFO Master manager not found
2019-11-20 17:00:23,423 file.LocalServiceAdmin INFO All the local services are enabled.
2019-11-20 17:00:23,424 file.LocalServiceAdmin INFO All the local services are enabled.
2019-11-20 17:00:23,446 file.manager INFO Component type conf file: Human_Tracking_py.conf loaded.
2019-11-20 17:00:23,459 file.ec_base INFO setRate(1000000.000000) done
2019-11-20 17:00:23,460 file.ec_base INFO setRate(100.000000) done
2019-11-20 17:00:23,463 file.rtobject INFO 1 execution context was created.
2019-11-20 17:00:23,476 file.rtobject INFO Initial active configuration set is default.
2019-11-20 17:00:30,565 file.SdoServiceAdmin WARNING No available SDO service in the factory: 
2019-11-20 17:00:30,566 file.SdoServiceAdmin ERROR type  already exists.
2019-11-20 17:00:35,592 file.ManagerServant INFO Master manager not found
2019-11-20 17:00:41,426 file.rtobject ERROR Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\OpenRTM_aist\RTObject.py", line 1571, in on_execute
    ret = self.onExecute(ec_id)
  File "c:\workspace\Human_Tracking_py\Human_Tracking_py.py", line 189, in onExecute
    self.x_seq=r_seq[44:726]*np.cos(th_seq)
ValueError: operands could not be broadcast together with shapes (682,) (681,) 

2019-11-20 17:00:47,855 file.ManagerServant INFO Master manager not found
2019-11-20 17:01:00,161 file.ManagerServant INFO Master manager not found
2019-11-20 17:01:05,033 file.ec_worker ERROR State of the RTC is not ACTIVE_STATE.
2019-11-20 17:01:05,053 file.ec_worker WARNING ExecutionContext is already stopped.
2019-11-20 17:01:05,055 file.ec_base ERROR Invoking on_shutdown() for each RTC failed.
